src/helpers/utils.ts

```diff
@@ -70,18 +70,23 @@ function isVariable(str: string) {
 
 function formatSection(section: ContextSummaryVariableType) {
   const varContext = section;
+  const units = varContext.units;
 
   if (varContext?.type === 'eth')
-    return `${utils.formatEther(varContext?.value)} ETH`;
+    return `${utils.formatEther(varContext?.value)} ETH${
```

Comment:
ETH would be the unit here
```suggestion
    return `${utils.formatEther(varContext?.value)} ${
```